### DISCUSSION TITLE
Cannot manage to set correc date/time on my yi dome

### DISCUSSION THREAD

**Discussion by ddcorazon** [roleoroleo/yi-hack-Allwinner-v2#889] [Q&A]
Hey y'all

Hope houre all doing well.

I started using the hack about 1 week ago and i cannot manage to have the correct time/date display (localtime or in the OSD time)

I'am using the Yi Dome Camera U (2K) and the hack h51ga

I set the correct time zone but it's not working. I live in Europe/Paris

![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/62076672/03025e24-083c-45f2-bf3f-d0bb3b3ead4d)

![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/62076672/1d104953-85fb-448d-8cbb-3ebadbc53950)
![image](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/62076672/0c23ccc0-4dc9-4e0e-8b45-016a8bddfae2)

**Comment by ddcorazon** [comment]
I found my problem. My bad. for all cameras i block the access to internet so it was not able to access to ntp servers.

**Reply by roleoroleo** [reply]
You can configure a local ntp server, if you want.
